authors/apps/notifications/cron_job.py

```diff
@@ -0,0 +1,60 @@
+from django_cron import CronJobBase, Schedule
+from django.conf import settings
+from django.template.loader import render_to_string
+from django.core.mail import EmailMessage
+
+from .models import Notification, CommentNotification
+
+
+class EmailNotificationCron(CronJobBase):
+    """Create the cron job for email sending."""
+
+    RUN_EVERY_MINS = settings.RUN_EVERY_MINS
+    schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
+    code = 'authors.apps.notifications.cron_job.EmailNotificationCron'  
+    
+    def do(self):
+        """Send emails to all the persons to be notified."""
```

Comment:
@zaabu Thanks for the work on this notifications feature, but I think this function would be more efficient if you reduced the code. You can try passing the model object Comment/Article as function variable to achieve it.

Reply:
Thanks, @kimbugp  for the feedback. I have reduced the code through the use of methods.